compass-custom-archetype/src/test/resources/projects/basic/archetype.properties

```diff
@@ -3,3 +3,4 @@ package=it.pkg
 version=0.1-SNAPSHOT
 groupId=archetype.it
 artifactId=basic
+compassVersion=1.0-SNAPSHOT
```

Comment:
Please clarify, why does this need to be here? And should it really be 1.0-SNAPSHOT?